### PR TITLE
Workflow submission file selection

### PR DESCRIPTION
**Pull request by joshsadam**
## What does this PR do and why?
_Describe in detail what your merge request does and why._

Currently, when a file is selected in the pipeline parameters dialog, if it is part of a pair, the second file is not selected.  This enhancement will automatically select the paired file in the second select dropdown.  If a single-ended file is selected, the second dropdown is cleared.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

image

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

1. Go to a project with samples (http://localhost:3000/yersinia/yersinia-pseudotuberculosis/outbreak-2023/-/samples)
2. Create a new sample and upload a paired-end set of files to it
3. Go back to project samples and select the new sample 
4. Select a pipeline and see how the file selection is updated.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [X] I have evaluated the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.

### CODE REVIEW

**Comment by deepsidhu85** [comment]
Found a bug. I uploaded a pair, selected it, launched the pipeline selection, then on the parameters portion the second file is selected but not the first

image

**Comment by joshsadam** [comment]
> Found a bug. I uploaded a pair, selected it, launched the pipeline selection, then on the parameters portion the second file is selected but not the first
> 
> image

I cannot reproduce this error, and I have tried multiple times now, can you try again and if it persists maybe we walk through what is happening together

**Review by deepsidhu85 — DISMISSED** [review]
Looks good to me!

File: app/components/nextflow/samplesheet/sample_cell_component.html.erb [hunk 1]
```diff
@@ -1,4 +1,4 @@
 <td class="px-6 py-3 text-left whitespace-nowrap">
   <%= sample.name %>
-  <%= fields.hidden_field :sample, value: "Sample_#{sample.id}" %>
+  <%= fields.hidden_field :sample, value: "Sample_#{sample.puid}" %>
```
Comment (ericenns) [inline app/components/nextflow/samplesheet/sample_cell_component.html.erb:3]: **nitpick (blocking):** The value should just be the puid, no prefix

**Review by ericenns — CHANGES_REQUESTED** [review]
One minor change

File: app/components/nextflow/samplesheet/sample_cell_component.html.erb (hunk 1 above)
Comment (joshsadam) [reply app/components/nextflow/samplesheet/sample_cell_component.html.erb]: Sorry, updated in 905a875

**Review by ericenns — APPROVED** [review]
Looks good to me!

**Pull request merged**